cypress/integration/e2e/table/schedule-modes.spec.ts

```diff
@@ -25,6 +25,17 @@ describe("Schedule modes spec", () => {
     });
   });
 
+  context("not saved changes are not visible", () => {
+    it("performs test", () => {
+      cy.getWorkerShift(testedCell);
+      cy.enterEditMode();
+      cy.changeWorkerShift({ ...testedCell, newShiftCode: testedCell.desiredShiftCode });
+      cy.get("[data-cy=leave-edit-mode]").click();
+      cy.get("[data-cy=bt-leave-edit-save-no]").click();
+      cy.checkWorkerShift({ ...testedCell, desiredShiftCode: testedCell.initialShiftCode });
```

Comment:
Zostawiłbym tylko tą linię w bloku `it`, resztę przenieść do bloku `context` powyżej.